core/src/operations/mod.rs

```diff
@@ -155,7 +155,7 @@ pub enum Operation {
 
     /// Pops two elements off the stack and checks if each of them represents a 32-bit value.
     /// If both of them are, they are pushed back onto the stack, otherwise an error is returned.
-    U32assert2,
+    U32assert2(Felt),
```

Comment:
We should explain what the internal values means. See how I did it on line 22 above for the simple assert.